climesync.py

```diff
@@ -170,6 +222,23 @@ def get_fields(fields):
     return responses
 
 
+def add_kv_pair(key, value, path="~/.climesyncrc"):
+    """Ask the user if they want to add a key/value pair to the config file"""
+
+    config = read_config(path)
+
+    # If that key/value pair is already in the config, skip asking
+    if config.has_option("climesync", key) \
+       and config.get("climesync", key) == value:
+        return
+
+    print "{} = {}".format(key, value)
+    response = get_field("Add to the config file?", field_type="?")
```

Comment:
@aaroncohen73 the uppercase `N` here implies that it is the default that is selected if you don't type anything
```
(y/N) Add to the config file?:
```
But I have to type `n` to select a no.